lib/screens/home_screen.dart

```diff
@@ -20,9 +14,24 @@ class HomeScreen extends StatefulWidget {
 }
 
 class _HomeScreenState extends State<HomeScreen> {
+  GlobalKey buttonGlobalKey = GlobalKey();
+  List<TBProductModel> products = [
+    product1,
+    product2,
+    product3,
+  ];
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      floatingActionButtonLocation: FloatingActionButtonLocation.centerDocked,
+      floatingActionButton: Padding(
+        padding: EdgeInsets.symmetric(
+            horizontal: TBDimensions.homeScreen.contentPadding),
+        child: TBButton(
+            key: buttonGlobalKey,
+            text: "Add product",
+            type: TBButtonType.filled),
+      ),
       backgroundColor: TBColor.app.backgroundColor,
       appBar: TBAppBar.styled(
```

Comment:
The app bar changes color on scrolling content. It should always be same color
<img width="475" alt="image" src="https://github.com/damjan-Igrowth/INT-Milan_Zezelj-Shop/assets/119666343/0b674660-2ead-40cc-b78c-d1cc4a6dc72e">
<img width="475" alt="image" src="https://github.com/damjan-Igrowth/INT-Milan_Zezelj-Shop/assets/119666343/d30ced76-6a06-4a0b-9199-188857f304b5">